apps/server/src/modules/school/api/test/school-users.api.spec.ts

```diff
@@ -340,112 +364,134 @@ describe('School Controller (API)', () => {
 			};
 
 			describe('user is student', () => {
-				it('should include students from current class', async () => {
-					const { loggedInClient, school, studentIdsOfCurrentClass } = await setup('student');
-
-					const response = await loggedInClient.get(`${school.id}/students`);
-					const body = response.body as SchoolUserListResponse;
-					const recievedIds = body.data.map((user) => user.id);
-
-					expect(recievedIds).toEqual(expect.arrayContaining(studentIdsOfCurrentClass.map((id) => id.toString())));
-				});
-
-				it('should not include students from archived class', async () => {
-					const { loggedInClient, school, studentIdsOfArchivedClass } = await setup('student');
-
-					const response = await loggedInClient.get(`${school.id}/students`);
-					const body = response.body as SchoolUserListResponse;
-					const recievedIds = body.data.map((user) => user.id);
-
-					expect(recievedIds).not.toEqual(expect.arrayContaining(studentIdsOfArchivedClass.map((id) => id.toString())));
-				});
-
-				it('should include students from moin.schule class', async () => {
-					const { loggedInClient, school, studentIdsOfMoinSchuleClass } = await setup('student');
-
-					const response = await loggedInClient.get(`${school.id}/students`);
-					const body = response.body as SchoolUserListResponse;
-					const recievedIds = body.data.map((user) => user.id);
-
-					expect(recievedIds).toEqual(expect.arrayContaining(studentIdsOfMoinSchuleClass.map((id) => id.toString())));
-				});
-
-				it('should not include students from archived moin.schule class', async () => {
-					const { loggedInClient, school, studentIdsOfArchivedMoinSchuleClass } = await setup('student');
-
-					const response = await loggedInClient.get(`${school.id}/students`);
-					const body = response.body as SchoolUserListResponse;
-					const recievedIds = body.data.map((user) => user.id);
-
-					expect(recievedIds).not.toEqual(expect.arrayContaining(studentIdsOfArchivedMoinSchuleClass.map((id) => id)));
+				describe('user is from same school', () => {
+					it('should include all students from the school', async () => {
```

Comment:
why is that test running fine if we just found out, that it does not work like that because of `STUDENT_LIST`?

Reply:
becasue its not doing what it says.. good catch :)